tests/getRevPagination.spec.js

```diff
@@ -0,0 +1,104 @@
+                    mockReviews({ verified: true })
+                ],
+                meta: {
+                    pagination: {
+                        page: 1,
+                        pageSize: 25,
+                        pageCount: 1,
+                        total: 3
+                    },
+                },
+            })
+        };
+
+        await getMovieReviews(3, cmsAdapterReviews)
+        request(app).get("/api/reviews/:movieId")
+            .query({ page: 1 })
+            .query({ limit: 5 })
+            .expect(200)
+            .then((res) => {
+                expect(res.body.length).toBeLessThanOrEqual(6);
+                expect(res.body.length).toBeGreaterThanOrEqual(1);
+            })
+    });
+
+    test('should return max 5 revies and 1 pagination data-object', async () => {
+        const cmsAdapterReviews = {
+            loadMovieReviews: async () => ({
+                data: [
+                    mockReviews({ verified: false }),
+                    mockReviews({ verified: null }),
+                    mockReviews({ verified: true }),
+                    mockReviews({ verified: false }),
+                    mockReviews({ verified: null }),
+                    mockReviews({ verified: true }),
+                    mockReviews({ verified: false }),
+                    mockReviews({ verified: null }),
+                    mockReviews({ verified: true }),
+                    mockReviews({ verified: false }),
+                    mockReviews({ verified: null }),
+                    mockReviews({ verified: true }),
+                    mockReviews({ verified: false }),
+                    mockReviews({ verified: null }),
+                    mockReviews({ verified: true }),
+                    mockReviews({ verified: false }),
+                    mockReviews({ verified: null }),
+                    mockReviews({ verified: true }),
+                    mockReviews({ verified: false }),
+                    mockReviews({ verified: null }),
+                    mockReviews({ verified: true }),
+                    mockReviews({ verified: false }),
+                    mockReviews({ verified: null }),
+                    mockReviews({ verified: false })
+                ],
+                meta: {
+                    pagination: {
+                        page: 1,
+                        pageSize: 24,
+                        pageCount: 1,
+                        total: 24
+                    },
+                },
+            })
+        };
+
+        await getMovieReviews(3, cmsAdapterReviews)
+        request(app).get("/api/reviews/:movieId")
+            .query({ page: 2 })
+            .query({ limit: 5 })
+            .expect(200)
+            .then((res) => {
+                expect(res.body.length).toBeLessThanOrEqual(6);
+                expect(res.body.length).toBeGreaterThanOrEqual(1);
+            })
+    });
+});
+
+function mockReviews(attributes) {
```

Comment:
The name of this function implies to me that it will mock multiple reviews (plural). I would have named it `mockReview()` instead. This sounds like a tiny thing (and it's not very big) but things like these do make code harder to read and reason about, when the names of things don't fully reflect what they do.